src/main/java/factionsystem/Commands/HelpCommand.java

```diff
@@ -60,15 +60,19 @@ public boolean sendHelpMessage(CommandSender sender, String[] args) {
                 sender.sendMessage(ChatColor.AQUA + "/mf swearfealty - Swear fealty to a faction.");
                 sender.sendMessage(ChatColor.AQUA + "/mf declareindependence - Declare independence from your liege.");
                 sender.sendMessage(ChatColor.AQUA + "/mf grantindependence - Grant a vassal vaction independence.");
-            }
-            if (args[1].equalsIgnoreCase("6")) {
-                sender.sendMessage(ChatColor.BOLD + "" + ChatColor.AQUA + "\n == Medieval Factions Commands Page 5/6 == " + "\n");
                 sender.sendMessage(ChatColor.AQUA + "/mf force - Force the plugin to perform certain actions." + "\n");
                 sender.sendMessage(ChatColor.AQUA + "/mf version - Check plugin version." + "\n");
                 sender.sendMessage(ChatColor.AQUA + "/mf resetpowerlevels - Reset player power records and faction cumulative power levels." + "\n");
+            }
+            if (args[1].equalsIgnoreCase("6")) {
+                sender.sendMessage(ChatColor.BOLD + "" + ChatColor.AQUA + "\n == Medieval Factions Commands Page 5/6 == " + "\n");
                 sender.sendMessage(ChatColor.AQUA + "/mf bypass - Bypass protections.");
                 sender.sendMessage(ChatColor.AQUA + "/mf config show - Show config values.");
                 sender.sendMessage(ChatColor.AQUA + "/mf config set (option) (value) - Set a config value.");
+		        sender.sendMessage(ChatColor.AQUA + "/mf gate name");
```

Comment:
Can we put these four new commands on page 5 of the help command instead of page 6?